YvesIraguha/personalFinanceMobile
# #171896067 Add Ability to Upload an Image

#### What does this PR do?
- refactor the codebase
- enable attaching an image on the investment
#### Description of Task to be completed?
- Replace hard-coded avatar with the avatar of the user 
- Fix the image rendering of items 
- Enable a user to upload an image of investment 
- Show the uploaded image to a user 
#### How should this be manually tested?
- Clone the repo and checkout the branch `ft-upload-images-#171896067`
- Install all dependencies by running `yarn add`
- Fill your`.env` file with correct credentials
- Start the backend server and the mobile app
- sign in and create an investment, you will see the ability to upload an image. 

#### What are the relevant pivotal tracker stories?
#171896067
